### ISSUE TITLE
Use a task runner to compile sass to css

### ISSUE THREAD

**Issue by rgupta-okta** [shawneetap/SoundCloud#4]
Right now you need to download an application to compile sass. While that works for you, if you want to push this code somewhere else, like a machine on aws, and have that machine in aws compile sass, then you want a command line utility to compile the sass. So the end result should be one script, whether it's in bash or grunt or node or any other technology, that compiles sass to css. 

There are a couple of ways to do this.
- the sass ruby gem (native sass support but it's ruby)
- PostCSS (yay javascript, but not super sass)
- libsass (C implementation of sass, there are node/grunt wrappers for this)

The reason I'm saying to do this is so that you get experience with real life build tools and making projects that others can easily consume. Look at things like npm, grunt, etc.

**Comment by shawneetap** [comment]
Alright! I used node and grunt to compile my sass to css. My Gruntfile.js is located within assets/grunt. Is this typical to have all of the grunt files local within a project? I also saw that there is a way to "watch" the sass files so I don't have to run the cmd every time I make a change to my sass. Im going to tackle that next. 

While I was committing the changes on the GitHub Desktop client, there were .sass cache files throwing errors and not allowing me to commit my changes, so I just unchecked them. Do I need those?

This is definitely the stuff I need to be learning. Keep it coming!
